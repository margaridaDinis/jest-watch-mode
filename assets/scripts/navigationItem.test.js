import navigationItem from './navigationItem';

describe('navigationItem', () => {
  it('removes href if edge page nav item', () => {
    expect(navigationItem(0)).toMatchSnapshot();
  });

  it('adds active if it matches the current page', () => {
    expect(navigationItem(1)).toMatchSnapshot();
  });

  it('does not add active nor removes edge for other cases', () => {
    expect(navigationItem(13)).toMatchSnapshot();
  });
});
