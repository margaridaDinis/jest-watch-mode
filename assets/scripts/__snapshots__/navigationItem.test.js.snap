// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigationItem adds active if it matches the current page 1`] = `
"
    <li class=\\"\\">
      <a href=\\"/?page=1\\">•</a>
    </li>
  "
`;

exports[`navigationItem does not add active nor removes edge for other cases 1`] = `
"
    <li class=\\"\\">
      <a href=\\"/?page=13\\">•</a>
    </li>
  "
`;

exports[`navigationItem removes href if edge page nav item 1`] = `
"
    <li class=\\"active\\">
      <a href=\\"\\">•</a>
    </li>
  "
`;
